### DISCUSSION TITLE
Subfinder API

### DISCUSSION THREAD

**Discussion by w4nnatry** [projectdiscovery/subfinder#887] [Q&A]
Subfinder v2.6.0
how to add API in config file?please give me example.

bevigil: []
binaryedge: []
bufferover: []
c99: []
censys: []
certspotter: []
chaos: []
chinaz: []
dnsdb: []
dnsrepo: []
fofa: []
fullhunt: []
github: []
hunter: []
intelx: []
leakix: []
netlas: []
passivetotal: []
quake: []
robtex: []
securitytrails: []
shodan: []
threatbook: []
virustotal: []
whoisxmlapi: []
zoomeye: []
zoomeyeapi: []

**Comment by ehsandeep** [comment]
@w4nnatry see the example - https://github.com/projectdiscovery/subfinder#post-installation-instructions

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
